modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPaste.ts

```diff
@@ -0,0 +1,143 @@
+import { RealClipboard, RealMouse } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyApis, TinyAssertions, TinyHooks, TinyUi } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+const hook = TinyHooks.bddSetup<Editor>({
+  base_url: '/project/tinymce/js/tinymce',
+  toolbar: false,
+  statusbar: false
+}, []);
+
+type Selection = [ number[], number, number[], number ];
+
+const copyAndPast = async (editor: Editor, source: Selection, target: Selection): Promise<void> => {
+  const ui = TinyUi(editor);
+  const api = TinyApis(editor);
+
+  api.setSelection(...source);
+  ui.clickOnMenu('button:contains("Edit")');
+  await ui.pWaitForUi('*[role="menu"]');
+  await RealMouse.pClickOn('div[title="Copy"]');
+  api.setSelection(...target);
+  await RealClipboard.pPaste('iframe => body => h1');
```

Comment:
```suggestion
  await RealClipboard.pPaste('iframe => body');
```
This doesn't look right, as `=>` means select across frames and there's not an iframe in the editor. Additionally, we don't have a `h1` element in all these tests.

Reply:
done